### ISSUE TITLE
Certificate issue after Tizen 8 One UI update

### ISSUE THREAD

**Issue by ruboncio** [Georift/install-jellyfin-tizen#51]
After Tizen 8 One UI update the app no longer works. I have tried running this again but it is failing on a certificate now. I've tried other guys where you sign the certificate with Tizen Studio and it doesn't work either. Any advise? Bloody Samsung, not sure why they can't just approve JellyFish so we don't have to keep doing this.

```
Platform log view
--------------------
install AprZAARz4r.Jellyfin
package_path /home/owner/share/tmp/sdk_tools/tmp/Jellyfin-TrueHD.wgt
app_id[AprZAARz4r.Jellyfin] install start
app_id[AprZAARz4r.Jellyfin] installing[9]
app_id[AprZAARz4r.Jellyfin] installing[19]
app_id[AprZAARz4r.Jellyfin] install failed[118, -12], reason: Check certificate error : :Invalid certificate chain with certificate in signature.:<-3>  
spend time for wascmd is [4624]ms
Failed to install Tizen application.
Total time: 00:01:18.811
```
![image](https://github.com/user-attachments/assets/ea99172e-1617-4f39-b3a0-22465b8fcc35)

**Comment by pyro-x** [comment]
I'm having exactly the same issue after the update

Transferred the package: /home/developer/Jellyfin-TrueHD.wgt -> /home/owner/share/tmp/sdk_tools/tmp
Installing the package...
--------------------
Platform log view
--------------------
install AprZAARz4r.Jellyfin
package_path /home/owner/share/tmp/sdk_tools/tmp/Jellyfin-TrueHD.wgt
app_id[AprZAARz4r.Jellyfin] install start
app_id[AprZAARz4r.Jellyfin] installing[9]
app_id[AprZAARz4r.Jellyfin] installing[19]
app_id[AprZAARz4r.Jellyfin] install failed[118, -12], reason: Check certificate error : :Invalid certificate chain with certificate in signature.:<-3>
spend time for wascmd is [6437]ms
Failed to install Tizen application.
Total time: 00:00:09.908

**Comment by rhuanlima** [comment]
I'm found a same issue here

```
install AprZAARz4r.Jellyfin
package_path /home/owner/share/tmp/sdk_tools/tmp/Jellyfin-TrueHD.wgt
app_id[AprZAARz4r.Jellyfin] install start
app_id[AprZAARz4r.Jellyfin] installing[9]
app_id[AprZAARz4r.Jellyfin] installing[19]
app_id[AprZAARz4r.Jellyfin] install failed[118, -12], reason: Check certificate error : :Invalid certificate chain with certificate in signature.:<-3>
spend time for wascmd is [6506]ms
Failed to install Tizen application.
Total time: 00:00:10.230
```

**Comment by rhuanlima** [comment]
This solution works for me

https://gist.github.com/SayantanRC/57762c8933f12a81501d8cd3cddb08e4

**Comment by realbaylen** [comment]
Having this issue as well

**Comment by Srippie1911** [comment]
Same here. Any response from the team?

**Comment by olanystrom** [comment]
Same issue here.

**Comment by CodeEtienne** [comment]
Same error here:
install failed[118, -12], reason: Check certificate error : :Invalid certificate chain with certificate in signature.:<-3>

**Comment by ryangrose** [comment]
+1

**Comment by rcbop** [comment]
+1

**Comment by Srippie1911** [comment]
Sam here

**Comment by jimx78** [comment]
Larspolo, quel est la solution stp ? Perso je ne trouve pas comment faire, j'ai comme tout le monde ce problème : 

> app_id[AprZAARz4r.Jellyfin] installing[19]
> app_id[AprZAARz4r.Jellyfin] install failed[118, -12], reason: Check certificate error : :Invalid certificate chain with certificate in signature.:<-3>
> spend time for wascmd is [7070]ms
> Failed to install Tizen application.
> Total time: 00:00:16.451

**Comment by vinyli85** [comment]
If you are on Tizen 8 OneUI you have to [create your own certificates using TizenStudio](https://developer.samsung.com/smarttv/develop/getting-started/setting-up-sdk/creating-certificates.html). During creation of your certificates you have to login into your Samsung account (same as on TV).

With this created certificates you can then install jellyfin on your tv using the [appropriate optional command ](https://github.com/Georift/install-jellyfin-tizen?tab=readme-ov-file#install-jellyfin)

**Comment by Epgenix** [comment]
@Georift Maybe i'm looking forward to simplify the process described [here](https://gist.github.com/SayantanRC/57762c8933f12a81501d8cd3cddb08e4) from @SayantanRC
Ill create a pull request on that to resolve this.

**Comment by maximemenager** [comment]
For those interested, it is possible to quickly generate certificates without installing Tizen Studio. I explain a solution here: https://github.com/Georift/install-jellyfin-tizen/issues/21#issuecomment-2614108372

**Comment by Boudewijn26** [comment]
For anyone struggling with the proposed solutions: be aware that the installer switched from docker hub to ghcr for the image hosting. You need to run the ghcr.io/georift/install-jellyfin-tizen not just the georift/install-jellyfin-tizen. (especially tenacious if you use autosuggestions like me)

**Comment by Epgenix** [comment]
Hey, I'm pretty busy at the moment. But I really want to move this project forward. Can someone who knows what they're doing help me with my current issue on my PR? #65

**Comment by Epgenix** [comment]
I managed to implement a automatic certificate generation and implement that into the installation process.
In the next days i am going to edit the readme file accordingly so there are new instructions to follor. But for all who are interested, you can take a look into my comments on #65. I exmplain there how to invoke the installation process. It's pretty simple.

**Comment by pazz** [comment]
@Epgenix : I tried your current main branch and your new argument seems not to be picked up.
The command I use is copied from your README with the obvious adjustments.
Thanks

```
docker run -p 4794:4794 ghcr.io/georift/install-jellyfin-tizen --ip MYIP --oneui8 --device-id GU43CU7179UXZG --email my@mail.com
basename: unrecognized option '--oneui8'
Try 'basename --help' for more information.
Error: Could not extract the tag from the provided URL --oneui8
Please provide a URL to the full release, for example: https://github.com/jeppevinkel/jellyfin-tizen-builds/releases/tag/2024-11-24-0431
Otherwise, don't provide a URL and the latest version will be installed.


```

**Comment by Epgenix** [comment]
[Check this out](https://github.com/Georift/install-jellyfin-tizen/pull/65#issuecomment-2676243914)

**Comment by pazz** [comment]
@Epgenix was this for me? 
I do not use docker every day but it looks to me as if the command i posted above picks a docker image hosted online, and not from my local checkout of your fork. All I'm saying is that the documentation is not as helpful as it could be. I for one do not know how to run your updated script from the README.md instructions. Thanks

**Comment by Epgenix** [comment]
Probably because the changes are notyet uploaded by @Georift i wrote these changes with the fact in mind that @Georift would merge it instantaneously (whitch not happened). Please refeer to the comments from #65.

**Comment by timopetric** [comment]
> Hey, you can follow my [gist](https://gist.github.com/dalvelab/b4470eb22d6eb0b45df16950a263e3e3)

This finally worked for me! Thanks for the clear instructions and the video.

**Comment by Epgenix** [comment]
> Hey, you can follow my [gist](https://gist.github.com/dalvelab/b4470eb22d6eb0b45df16950a263e3e3)

Why simple when it also can be done in a complex way 😂 
Sorry, but i already provided a solution that is automated and works fine. Sure it has some wierd bugs. But encountering them is basically just a thing of: "Have you read the readme file correctly or not". 

I am just waiting for @Georift to merge my stuff and upload the docker container to the repo. If that is done, everything will work om the first try, under the condition that you have the correct unique device id. If not then you will encounter the certificate error from the script. But like i said. It's a thing of, if you hav read the readme file accordingly.

Basically it's the choice of everyone here how they want to install it. I'm not trying to blame on someone here. Have fun with installing.

Cheers.

**Comment by PatrickSt1991** [comment]
Maybe not the best place to put this, but
[Jellyfin 2 Samsung](https://github.com/PatrickSt1991/Samsung-Jellyfin-Installer) is a one click Windows application, right now the app is in beta for Tizen 7+ but after yesterday's update it should be able to deploy to the newer Tizen TVs

**Comment by piyanutwat** [comment]
> After Tizen 8 One UI update the app no longer works. I have tried running this again but it is failing on a certificate now. I've tried other guys where you sign the certificate with Tizen Studio and it doesn't work either. Any advise? Bloody Samsung, not sure why they can't just approve JellyFish so we don't have to keep doing this.
> 
> ```
> Platform log view
> --------------------
> install AprZAARz4r.Jellyfin
> package_path /home/owner/share/tmp/sdk_tools/tmp/Jellyfin-TrueHD.wgt
> app_id[AprZAARz4r.Jellyfin] install start
> app_id[AprZAARz4r.Jellyfin] installing[9]
> app_id[AprZAARz4r.Jellyfin] installing[19]
> app_id[AprZAARz4r.Jellyfin] install failed[118, -12], reason: Check certificate error : :Invalid certificate chain with certificate in signature.:<-3>  
> spend time for wascmd is [4624]ms
> Failed to install Tizen application.
> Total time: 00:01:18.811
> ```
> ![image](https://github.com/user-attachments/assets/ea99172e-1617-4f39-b3a0-22465b8fcc35)
>
